### PR TITLE
Manage Organisation doesn't work on IE11

### PR DESCRIPTION
**Pull request by RiteshHMCTS**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3510

### Change description ###
Manage Organisation doesn't work on IE11 browser

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by phillip-whitaker** [comment]
Out of curiosity.. why do these libraries solve the problem?

**Comment by paul-graham** [comment]
> Out of curiosity.. why do these libraries solve the problem?

Beat me to it, @phillip-whitaker! Same question...

**Review by leepavlouhmcts — APPROVED** [review]
looks good

**Comment by RiteshHMCTS** [comment]
> Out of curiosity.. why do these libraries solve the problem?

I have added isomorphic-fetch which was used to solve the same issue in MC.

File: package.json [hunk 1]
```diff
@@ -210,6 +211,8 @@
     "@stryker-mutator/karma-runner": "^3.0.1",
     "@stryker-mutator/mocha-runner": "^3.0.1",
     "@stryker-mutator/typescript": "^3.0.1",
-    "**/**/ini": "^1.3.6"
+    "**/**/ini": "^1.3.6",
+    "**/**/prismjs": "^1.23.0",
+    "**/**/serialize-javascript": "^3.1.0"
```
Comment (phillip-whitaker) [inline package.json:216]: What's with these additions? It is not on the other PR with the same purpose

Comment (RiteshHMCTS) [reply package.json:216]: these are security vulnerability issue which popped up today morning

**Comment by phillip-whitaker** [comment]
> > Out of curiosity.. why do these libraries solve the problem?
> 
> I have added isomorphic-fetch which was used to solve the same issue in MC.

I was more asking if we knew "why" it fixes the problem. I don't mind approving but for my understanding I wanted to make sure we were not just throwing libraries in. I have since found the information: https://github.com/matthew-andrews/isomorphic-fetch @paul-graham "This module will enable you to use fetch in your Node code in a cross-browser compliant fashion. "

**Review by phillip-whitaker — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (phillip-whitaker) [reply package.json:216]: I haven't seen this syntax before, is this pinning the version of prismjs that is a sub-dependency to top-level dependencies to a specific version?

Comment (RiteshHMCTS) [reply package.json:216]: https://www.npmjs.com/advisories/1548
https://www.npmjs.com/advisories/1638

**Pull request merged**